packages/story-utils/src/getStories.ts

```diff
@@ -10,13 +11,21 @@ type StoryData = {
   storyFn: Story<unknown>;
 };
 
+const metadataSchema = zod
```

Comment:
We can generate typescript types from these schemas. For this one it would look like

```ts
type Metadata = zod.infer<typeof metadataSchema>;
```